wrapanapi/systems/openstack.py

```diff
@@ -63,7 +63,41 @@ def _request_timeout_handler(self, url, method, retry_count=0, **kwargs):
             return self.request(url, method, retry_count=retry_count, **kwargs)
 
 
-class OpenstackInstance(Instance):
+class _SharedMethodsMixin(object):
+    """
+    Mixin class that holds properties/methods both VM's and templates share.
+
+    This should be listed first in the child class inheritance to satisfy
+    the methods required by the Vm/Template abstract base class
+    """
+    @property
+    def _identifying_attrs(self):
+        return {'uuid': self._uuid}
```

Comment:
it would be better to use public property instead of private one

Reply:
the abstract is 'private', so its consistent here. We can address this separately.